Plots/Bar/NCL_bar_7.py

```diff
@@ -0,0 +1,125 @@
+"""
+
+###############################################################################
+# Import the necessary python libraries
+import matplotlib.pyplot as plt
+from matplotlib.ticker import MultipleLocator
+
+
+###############################################################################
+# Create the plot data
+x = [1, 2, 3, 4, 5, 6, 7, 8]
+data = [154900, 56600, 40000, 30200, 29700, 24400, 21700, 13900]
+labels = ['Lung', 'Colon/rectum', 'Breast', 'Prostate', 'Pancreas',  
+          'Non-Hodgkin\'s Lymphoma', 'Leukemias', 'Ovary']
+
+###############################################################################
+# Create the custom color list.
+color_list = ['firebrick', 'red', 'orange', 'green', 'navy', 'blue', 'skyblue', 'slateblue']
+
+###############################################################################
+# Specify some plot settings.
+
+# Title settings
+title = 'Estimated Cancer Deaths for 2002'
+title_fontsize = 18
+
+# Axis Settings
+plot_y_max = 180_000
+
+# Tick Settings
+major_tick_spacing = 30_000
+minor_tick_spacing = 10_000
+tick_label_fontsize = 12
+tick_length_multiplier = 2
+
+# Label Settings
+label_rotation = 45
+label_y_offset = 2000
+
+###############################################################################
+# Create the first bar chart.
+
+# Figure size is (width, height) inches.
+plt.figure(1, figsize=(6, 6))
+
+plt.bar(x, data, color=color_list, edgecolor='black')
+plt.title(title, fontsize=title_fontsize)
+
+# Add a rotated label to each bar.
+for k, label in enumerate(labels):
+    plt.text(x[k], data[k]+label_y_offset, label, rotation=label_rotation)
+
+
+# Draw ticks on three sides of the plot.  Suppress tick labels on the bottom.
+plt.tick_params(which='both', top=True, right=True, left=True, bottom=False, labelsize=tick_label_fontsize)
+plt.xticks([], [])
+
+# Set the tick spacing and limits for the Y axis.
+ax = plt.gca()
+ax.yaxis.set_major_locator(MultipleLocator(major_tick_spacing))
+ax.yaxis.set_minor_locator(MultipleLocator(minor_tick_spacing))
+
+# Increase the tick mark lengths by some factor.
+y_major_tick_length = plt.rcParams["ytick.major.size"]
+y_minor_tick_length = plt.rcParams["ytick.minor.size"]
+plt.tick_params(which='major', length=y_major_tick_length * tick_length_multiplier)
+plt.tick_params(which='minor', length=y_minor_tick_length * tick_length_multiplier)
+
+# Set the limits for the Y axis.
+plt.ylim(top=plot_y_max)
+
+# Draw plot on the screen.
+plt.show()
+
+###############################################################################
+# Create the second bar chart with a legend.
+
+
+## NOTE: you may need to close the first figure window to see the second figure.
+# Figure size is (width, height) inches.
+plt.figure(2, figsize=(6, 6))
+
+bar_handle = plt.bar(x, data, color=color_list, edgecolor='black')
+plt.ylabel("Number of Deaths", fontsize = 16)
```

Comment:
For some reason Y axis label doesn't appear to be showing up. Do you think you could figure out why?

Reply:
@clyne  The script creates two plots, where the first does not have a Y axis label, and the second one does (as in the NCL examples).   Can you confirm that the Y axis label is not showing up on the second plot?

Reply:
I just confirmed that the Y axis label does not show up. @khallock, @erogluorhan can you please see if you can reproduce this problem? The NCL plot contains a Y-axis label and should look like this:

https://www.ncl.ucar.edu/Applications/Images/bar_7_2_lg.png

Reply:
Yes, I also confirm that Y axis label does not show up on Geocat-Examples webpage. I think that this is due to the aspect ratio management of the Geocat-examples webpage rather than the source codes because you can also see in the below screenshot that the Y-axis of my `NCL_xy_2` plot shows up as half-cut:

![image](https://user-images.githubusercontent.com/32553057/73289469-fc8bd880-41b9-11ea-80eb-b94bdecc8485.png)

@khallock Could you please have a look at this?